frontend/packages/kubevirt-plugin/src/components/modals/descheduler-modal/descheduler-modal.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import { Alert, Checkbox } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import {
+  createModalLauncher,
+  ModalBody,
+  ModalComponentProps,
+  ModalTitle,
+} from '@console/internal/components/factory';
+import './descheduler-modal.scss';
+import { ModalFooter } from '../modal/modal-footer';
+
+type DeschedulerModalProps = {
+  isVMdeschedulerOn: boolean;
+} & ModalComponentProps;
+
+const DeschedulerModal: React.FC<DeschedulerModalProps> = ({ isVMdeschedulerOn, close }) => {
+  const { t } = useTranslation();
+  const [isOn, setOn] = React.useState(isVMdeschedulerOn); // the default is OFF, the admin has to opt-in this feature
+  const submit = (event) => {
+    event.preventDefault();
+    // TODO, save the annotation, or remove it, according to the state
+    close();
+  };
+  const deschedulerChange = (checked) => {
+    setOn(checked);
+  };
+
+  return (
+    <div className="modal-content modal-content--no-inner-scroll">
+      <ModalTitle>{t('kubevirt-plugin~Descheduler settings')}</ModalTitle>
+      <ModalBody>
+        <Checkbox
+          id="descheduler-checkbox"
+          isChecked={isOn}
+          data-checked-state={isOn}
```

Comment:
what is this used for? I'm not familiar with this prop

Reply:
not sure about that, too, removed that